cpp/arcticdb/version/version_map_batch_methods.hpp

```diff
@@ -117,6 +120,130 @@ inline std::shared_ptr<std::unordered_map<std::pair<StreamId, VersionId>, AtomKe
     return output;
 }
 
+struct StreamVersionData {
+    explicit StreamVersionData(const pipelines::VersionQuery& version_query) {
+        react(version_query);
+    }
+
+    StreamVersionData() = default;
+
+    size_t count_ = 0;
+    LoadParameter load_param_ = LoadParameter{LoadType::LOAD_LATEST_UNDELETED};
```

Comment:
Are those supposed to be used outside of the struct? Should we make them private?

Reply:
tests shouldn't change the structure of the code in my opinion. Is there another way we can structure the tests?

Reply:
I think if we do that then we probably need to move StreamVersionData to its own files, otherwise it either looks like it's part of the API of the stream version methods (which it isn't), or becomes impossible to test.